src/anemoi/training/data/scaling.py

```diff
@@ -77,3 +81,59 @@ def scaler(plev: float) -> np.ndarray:
         del plev  # unused
         # no scaling, always return 1.0
         return 1.0
+
+
+class BaseAreaWeights:
+    """Method for overwriting the area-weights stored in the graph object."""
+
+    def __init__(self, target_nodes: str):
+        """Initialize area weights with target nodes.
+
+        Parameters
+        ----------
+        target_nodes : str
+            Name of the set of nodes to be rescaled (defined when creating the graph).
+        """
+        self.target = target_nodes
+
+    def area_weights(self, graph_data: HeteroData) -> torch.Tensor:
+        return torch.from_numpy(self.global_area_weights(graph_data))
+
+    def global_area_weights(self, graph_data: HeteroData) -> np.ndarray:
```

Comment:
The intention was not to have BaseAreaWeights as a default in forecaster to always re-calculate the areaweights in the graph. The default in forecaster is still to load the area-weights from the graph, which is achieved by not specifying training.loss_scaling.spatial in the config. 
I included this because it seemed like an appropriate base class for the cutout-reweighing (and potentially others to come), and for the very limit use-case when someone has made a graph with (for example) uniform weights and wish to overwrite those.